src/kontrol/kdist/foundry.md

```diff
@@ -608,7 +535,7 @@ All cheat code calls which take place while `expectRevert` is active are ignored
       [priority(35)]
 ```
 
-The `#halt` production is used to examine the end of each call in KEVM.
+The `#halt` production is used to examine the end of each call in Kontrol.
```

Comment:
Here, `KEVM` is not used as the tool's name, but as the semantics on top of which Kontrol is defined. Basically, the line says that we're examining the end of each call in `evm-semantics`.